plugins/kubesvc/patch-clouddriver.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    profiles:
+      clouddriver:
+        kubernetes:
+          enabled: true
+
+        sql:
```

Comment:
Shouldn't that part be in a separate `sql/` directory?

Reply:
There's already a config for just sql. The intent here was to have a better user experience by having everything a feature needs to have in the least amount of separate files, so that instead of telling users to add patches x, y and z for kubesvc to work, we can tell them to just add x and it's done.

Reply:
We'd have 2 places configuring the same thing, it has to be confusing. Which one gets priority? You could try to include the other kustomization here. Not sure what happens if the same kustomization is included twice.

Reply:
I think that would be a problem. I'll remove sql from here since there's nothing specific for kubesvc, and I'll add notes to indicate that other file is a requirement